src/utils/apiPaths.ts

```diff
@@ -0,0 +1,14 @@
+import { decamelizeKeys } from 'humps';
```

Comment:
nit: this fine here. Probably a bit better to have this file live in the api directory